### ISSUE TITLE
raspbee out of production. List of compatible modules

### ISSUE THREAD

**Issue by umibozu83** [diyhue/diyHue#83]
I read on the producers' website that the raspbee is out of production. Can I have a list of compatible products to use with diyhue?

**Comment by Mevel** [comment]
You can use conbee! Same producer but USB dongle. Works great!

**Comment by betonishard** [comment]
Or may-bee add support for zigbee2mqtt?

**Comment by mariusmotea** [comment]
I don't see a big support on issues from zigbee2mqtt and currently i'm involved in too many activities.

**Comment by fallenruby** [comment]
You should really consider adding support to zigbee2mqtt, since a CC2531 USB stick cost around 5$.

**Comment by cheesemarathon** [comment]
zigbee2mqtt stats:
[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/Koenkk/zigbee2mqtt.svg)](http://isitmaintained.com/project/Koenkk/zigbee2mqtt "Average time to resolve an issue")
[![Percentage of issues still open](http://isitmaintained.com/badge/open/Koenkk/zigbee2mqtt.svg)](http://isitmaintained.com/project/Koenkk/zigbee2mqtt "Percentage of issues still open")
[![Commit activity](https://img.shields.io/github/commit-activity/y/Koenkk/zigbee2mqtt.svg)](https://img.shields.io/github/commit-activity/y/Koenkk/zigbee2mqtt.svg "Commit activity")

diyHue stats:
[![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/diyhue/diyHue.svg)](http://isitmaintained.com/project/diyhue/diyHue "Average time to resolve an issue")
[![Percentage of issues still open](http://isitmaintained.com/badge/open/diyhue/diyHue.svg)](http://isitmaintained.com/project/diyhue/diyHue "Percentage of issues still open")
[![Commit activity](https://img.shields.io/github/commit-activity/y/diyhue/diyHue.svg)](https://img.shields.io/github/commit-activity/y/diyhue/diyHue.svg "Commit activity")

Yes, zigbee2mqtt is worse at closing it's issue, but it's had alot more of them than this project. It also has a higher commit rate and more contributors. So I would hardly say the project is unmaintained or slow. I think zigbee2mqtt is something worth looking at in the future, but as we have a working zigbee system already, it is not a priority. 

TLDR: You can use Conbee to replace Raspbee. zigbee2mqtt support may be added in the future.

**Comment by cheesemarathon** [comment]
I'm closing this. It's been added to the suggested enhancements project so we remember to work on it in the future.

**Comment by timbru31** [comment]
@umibozu83 - could you please post a link with the source that the RaspBee module is out of production? I couldn't find any source for that?

**Comment by manup** [comment]
The RaspBee should be available again soon on Amazon, it's definitely still in production.
Only ConBee I was replaced by ConBee II.

**Comment by timbru31** [comment]
Thanks for clarification, I assumed the original author meant that a new ConBee II is released :) 
@mariusmotea should we unpin this issue?

**Comment by mariusmotea** [comment]
As @manup said, RaspBee is still in production so issue here. I believe the changes made to Conbee 2 will not be visible to diyhue, probably is just a better ZigBee implementation...
